modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/DataSets.tsx

```diff
@@ -675,7 +675,7 @@ const DataSets = ({
 	];
 
 	return (
-		<div className="data-sets">
+		<div className="custom-data-sets data-sets">
```

Comment:
Add specific className to differentiate between `custom` and `system` but keeping `data-sets` to prevent break locators and styles.